tests/php/test-class-amp-core-block-handler.php

```diff
@@ -29,6 +33,7 @@ public function setUp() {
 			$this->markTestSkipped( 'Missing required render_block filter.' );
 		}
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).